### PR TITLE
Feature/sections

### PR DESCRIPTION
**Pull request by rasmusebbesen**

### CODE REVIEW

File: Sources/NStack/Translate/Translation.swift [hunk 1]
```diff
@@ -60,10 +60,24 @@ public struct Translation {
         }
     }
     
+    func getSection(section: String) -> Node {
```
Comment (steffendsommer) [inline Sources/NStack/Translate/Translation.swift]: Should we align this (with above) and just define it as `get(section: String) -> Node`.

File: Sources/NStack/Translate/Translate.swift [hunk 2]
```diff
@@ -81,6 +81,16 @@ public final class Translate {
         return get(platform: config.defaultPlatform, language: language, section: section, key: key, replace: replace)
     }
     
+    public final func getSection(section: String) throws -> Node {
```
Comment (steffendsommer) [inline Sources/NStack/Translate/Translate.swift]: I think we should align the naming, e.g. `get(section: String) throws -> ...`
Thinking about it - how do we handle the replace values here?

File: Sources/NStack/Translate/Translate.swift [hunk 3]
```diff
@@ -81,6 +81,16 @@ public final class Translate {
...
+        application.nStackConfig.log("Requesting translate for platform: \(config.defaultPlatform) - language: \(config.defaultLanguage) - section: \(section)")
+        let translate: Translation = try self.fetch(
```
Comment (steffendsommer) [inline Sources/NStack/Translate/Translate.swift]: Do we need to catch the errors here (looking at aline 51)?

Comment (rasmusebbesen) [reply Sources/NStack/Translate/Translate.swift]: done

Comment (rasmusebbesen) [reply Sources/NStack/Translate/Translate.swift]: I am not sure... the fallback in the original case is the key as a string, displayable in the views, where this is returning a node. We're not predicting the keys in the given section here. We're retrieving it all. I don't know what the fall back should be in this case.

File: Sources/NStack/Translate/Translation.swift [hunk 4]
```diff
@@ -60,10 +60,24 @@ public struct Translation {
...
+            return Translation.fallback(section: section)
+        }
+    }
+
     public static func fallback(section: String, key: String) -> String{
         return section + "." + key
     }
-    
+
+    public static func fallback(section: String) -> Node{
```
Comment (steffendsommer) [inline Sources/NStack/Translate/Translation.swift]: `Node {` (missing space)

File: Sources/NStack/Translate/Translate.swift (hunk 3 above)
Comment (steffendsommer) [reply Sources/NStack/Translate/Translate.swift]: Ye I agree - let's make it throw. This case is a bit harder to recover from.

**Pull request merged**